lib/components/viewers/nearby/rental-station.tsx

```diff
@@ -0,0 +1,69 @@
+import { Bicycle } from '@styled-icons/fa-solid/Bicycle'
+import { Company } from '@opentripplanner/types'
+import { connect } from 'react-redux'
+import { Parking } from '@styled-icons/fa-solid/Parking'
+import React from 'react'
+
+import * as mapActions from '../../../actions/map'
+import { FormattedMessage } from 'react-intl'
+import { IconWithText } from '../../util/styledIcon'
+
+import { Card, CardBody, CardHeader, CardSubheader } from './styled'
+
+import { useMap } from 'react-map-gl'
+
+type Props = {
+  companies: Company[]
+  place: any
+  zoomToPlace: (map: any, stopData: any) => void
+}
+const RentalStation = ({ companies, place, zoomToPlace }: Props) => {
+  const map = useMap().default
+  const { networks } = place
+  const network = networks.length === 1 ? networks[0] : null
+  const company = companies.find((c) => c.id === network)?.label
+  const { bikesAvailable, spacesAvailable } = place
+  return (
+    <Card onMouseEnter={() => zoomToPlace(map, place)}>
+      <CardHeader>
+        {place.name}
+        <CardSubheader>
+          {company || (
+            <FormattedMessage id="components.NearbyView.bikeRentalStation" />
+          )}
+        </CardSubheader>
```

Comment:
shouldn't the company come first?

Reply:
I don't know what you mean? I think this is right